angularjs-portal-frame/src/main/webapp/partials/search.html

```diff
@@ -1,6 +1,26 @@
 <form ng-submit="submit()">
-  <div class="input-group">
-    <input type="search" class="main-search" placeholder="Search MyUW" name="initialFilter" ng-model="initialFilter" aria-label="Search MyUW" focus-me="showSearchFocus">
+  <div class="input-group portlet-search-bar">
```

Comment:
We may just want to overwrite the frame search in the home module as this is pretty MyUW specific.